kyuubi-server/src/main/scala/org/apache/kyuubi/server/trino/api/KyuubiTrinoOperationTranslator.scala

```diff
@@ -0,0 +1,47 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.kyuubi.server.trino.api
+
+import org.apache.hive.service.rpc.thrift.TProtocolVersion
+
+import org.apache.kyuubi.operation.OperationHandle
+import org.apache.kyuubi.service.BackendService
+import org.apache.kyuubi.session.KyuubiSessionManager
+import org.apache.kyuubi.sql.trino.TrinoGetSchemas
+
+class KyuubiTrinoOperationTranslator(backendService: BackendService) {
+  lazy val parser = backendService.sessionManager.asInstanceOf[KyuubiSessionManager].parser
+
+  def transform(
+      statement: String,
+      user: String,
+      ipAddress: String,
+      configs: Map[String, String]): OperationHandle = {
+    val sessionHandle = backendService.openSession(
+      TProtocolVersion.HIVE_CLI_SERVICE_PROTOCOL_V11,
+      user,
+      "",
+      ipAddress,
+      configs)
+    parser.parsePlan(statement) match {
+      case TrinoGetSchemas(catalogName, schemaPattern) =>
+        backendService.getSchemas(sessionHandle, catalogName, schemaPattern)
+      case _ => throw new UnsupportedOperationException()
```

Comment:
Here maybe should have a `PassThrough` for execute statement (be.executeStatement)

Reply:
yes, we should have a PassThrough here

Reply:
Maybe need to separate the kyuubi trino fe parser file, which only needs case trino compatibility nodes, such as `TrinoGetSchemas`, and `PassThrough` for unmatched syntax, `UnsupportedOperationException` is useless.

Or, make a compatibility node like `TrinoGetSchemas` inherit `KyuubiTrinoFeNode`, translate once `KyuubiTrinoFeNode` is matched, and `PassThrough` all the other nodes. Otherwise SQL like 'KYUUBI SHOW SESSIONS' will throw unsupported exceptions(current parser will parse to TrinoGetSchemas, PassThrough, and RunnableCommand)